tests/ui/specs/VirtualList/VirtualGridList/qa-VirtualGridList/VirtualGridList-GT-28982-specs.js

```diff
@@ -0,0 +1,46 @@
+const Page = require('../VirtualGridListPage');
+const {expectFocusedItem} = require('../../VirtualList-utils');
+
+describe('qa-VirtualGridList translate mode', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should spotlight is on the bottom/top in same column via page up/down key [GT-28982]', function () {
+		// Step 2: Uncheck the "Native Scrolling" CheckboxItem.
+		Page.buttonModeChange.moveTo();
```

Comment:
I wonder, there is same TC for native scroll. 
If not, how about copy code and attach below on this file for native?

Reply:
Sugestted chagne.